spec/spec_helper_acceptance.rb

```diff
@@ -1,8 +1,12 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
 configure_beaker do |host|
-  if fact_on(host, 'osfamily') == 'RedHat' && fact_on(host, 'operatingsystemmajrelease') == '7'
+  if fact_on(host, 'osfamily') == 'RedHat' && fact_on(host, 'operatingsystemmajrelease').to_i == 7
     install_module_from_forge_on(host, 'puppet/epel', '>= 3.0.0')
   end
-  host.install_package('puppet-bolt')
+  unless fact_on(host, 'osfamily') == 'RedHat' && fact_on(host, 'operatingsystemmajrelease').to_i >= 9
+    # puppet-bolt rpm for CentOS 9 is not yet available
+    # https://tickets.puppetlabs.com/browse/MODULES-11275
+    host.install_package('puppet-bolt')
+  end
```

Comment:
I don't have any clear solution about this, mainly because the first clause is "if" and the second clause is "unless"

We might be able to marge them by osfamily layer but that requires adding a separate block for different distros so it adds additional lines.

Reply:
Ah you're right. While I was working on this I also considered using `os = fact_on(host, 'os')` and then using it as hash but I'd be OK with that as a future improvement.